coral/src/app/components/PreviewBanner.tsx

```diff
@@ -0,0 +1,54 @@
+import { Box, Flexbox, Icon, Typography } from "@aivenio/design-system";
+import infoSign from "@aivenio/design-system/dist/module/icons/infoSign";
+
+function Link({
+  text,
+  target,
+  isRemote,
+}: {
+  text: string;
+  target: string;
+  isRemote: boolean;
+}) {
+  const remoteAttrs = isRemote && {
+    target: "_blank",
+    rel: "noreferrer",
+  };
+  return (
+    <a href={target} {...remoteAttrs}>
+      <Typography color="primary-80" variant={"body-small"} htmlTag="span">
+        {text}
+      </Typography>
+    </a>
+  );
+}
+
+function PreviewBanner({ linkTarget }: { linkTarget: string }) {
+  return (
+    <Box backgroundColor={"info-5"} padding={"l1"}>
```

Comment:
`role="region"` in an easy add, but `id` property is not supported for `<Typography />`. Do you think it makes sense to wrap the whole thing in a `div` to make it accessible?

Reply:
Ah, true, I've had that problem, too. 💡  You could wrap the text in the `<Typography />` in a `<span>` and give that the id. It makes the html output a bit less pretty, but has no downsides otherwise.